netci.groovy

```diff
@@ -120,45 +130,6 @@ platformList.each { platform ->
     Utilities.addArchival(newJob, settings)
 }
 
-// **************************
```

Comment:
Not related to this change, but I wonder why the Linux arm32 and arm64 docker containers are different from the ones we use in netci.groovy for dotnet/coreclr.

Reply:
I had to update the arm64 containers to get cross crossgen working last spring.  I don't remember the details, but I might not have updated the coreclr containers to match...